torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -789,6 +789,13 @@ AOTI_TORCH_EXPORT AOTITorchError aoti_torch_view_dtype(
   });
 }
 
+AOTI_TORCH_EXPORT void aoti_torch_print_tensor_handle(
+    AtenTensorHandle self,
+    const char* msg) {
+  at::Tensor* t = reinterpret_cast<at::Tensor*>(self);
+  std::cout << "[" << msg << "]:" << *t << "\n";
```

Comment:
If `msg` is nullptr, the text after `msg` will be dropped.

Reply:
Fixed. Thanks.